src/main/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/messages.properties

```diff
@@ -8,6 +8,8 @@ line.new=''{0}'' should be on a new line.
 line.previous=''{0}'' should be on the previous line.
 no.line.wrap={0} statement should not be line-wrapped.
 single.space.separator=Use a single space to separate non-whitespace characters.
+too.many.single.line=''{0}'' method calls on single line, max allowed is ''{1}''.
```

Comment:
> '2' method calls on single line, max allowed is '1'.

Is it normal we put `'` around just numbers? It seems odd.
https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/com/puppycrawl/tools/checkstyle/checks/sizes/messages.properties
Nothing else in sizes does this.

It also looks like our basic size violation is "Count is {X} (max allowed is {Y}".

Reply:
Done.

Reply:
````
too.many.single.line={0,number,integer} method calls on single line, max allowed is {1,number,integer}.
wrap.method.call=Wrap chained method call, max allowed on a line is {0,number,integer}.
````
==>
````
too.many.single.line={0,number,integer} method calls on single line (max allowed is {1,number,integer})
wrap.method.call={0,number,integer} wrap chained method calls on a single line (max allowed is {0,number,integer})
````

Please confirm wording on `wrap.method.call` is correct.

Reply:
```
wrap.method.call={0,number,integer} wrap chained method calls on a single line (max allowed is {0,number,integer})
```
This one doesn't quite work like this. It would flag every method call on the line that is after the limit, for eg:
```xml
<module name="ChainedMethodCallWrap">                   
  <property name="identifierPattern" value="^method.*"/>
  <property name="maxCallsPerLine" value="1"/>          
</module>                                               
```
```java
public class Test {
    public static void main(String[] args) {
        method().
            method1().method2().method3();
    }
}
```
```bash
$ java -jar checkstyle-10.7.0-SNAPSHOT-all.jar -c config.xml Test.java 
Starting audit...
[WARN] Test.java:4:30: Wrap chained method call, max allowed on a line is 1. [ChainedMethodCallWrap]
[WARN] Test.java:4:40: Wrap chained method call, max allowed on a line is 1. [ChainedMethodCallWrap]
Audit done.
```
So both `method2()` and `method3()` are flagged.

Reply:
Are you agreeing with my change to `too.many.single.line` and only `wrap.method.call` is at issue?

Reply:
Done. Changed `too.many.single.line`.

Reply:
@Vyom-Yadav So what is the issue with changing the other?